### PR TITLE
Override strict_variable option using config

### PR DESCRIPTION
**Pull request by bernard-ng**
override the mode '**strict_variables**' in the **TwigEnvironmentFactory** via the configuration

### CODE REVIEW

**Review by michalbundyra — CHANGES_REQUESTED** [review]
LGTM but we need update also README.md, please

**Comment by froschdesign** [comment]
> …we need update also README.md

The readme file is maybe correct here, but the documentation is always more important!

**Comment by bernard-ng** [comment]
still waiting for a merge

**Review by geerteltink — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -133,6 +133,7 @@ Optional configuration can be stored in `config/autoload/templates.global.php`.
     'optimizations' => -1, // -1: Enable all (default), 0: disable optimizations
     'autoescape' => 'html', // Auto-escaping strategy [html|js|css|url|false]
     'auto_reload' => true, // Recompile the template whenever the source code changes
+    'strict_variables' => false // Ignore invalid variables
```
Comment (michalbundyra) [inline README.md]: trailing comma, please

**Review by michalbundyra — APPROVED** [review]
@bernard-ng The PR looks good for me, but as @froschdesign we need also updated documentation, please see: https://github.com/zendframework/zend-expressive/blob/master/docs/book/v3/features/template/twig.md
So we need another PR to that library with changes in that file.

**Pull request closed without merging**